enterprise/frontend/src/embedding-sdk/lib/usage-problem.ts

```diff
@@ -0,0 +1,95 @@
+import { match } from "ts-pattern";
+import { t } from "ttag";
+
+import type { SDKConfig } from "embedding-sdk/types";
+import type { SdkUsageProblem } from "embedding-sdk/types/usage-problem";
+
+import { getIsLocalhost } from "./is-localhost";
+
+interface SdkProblemOptions {
+  config: SDKConfig;
+
+  isEnabled: boolean;
+  hasTokenFeature: boolean;
+}
+
+const PROBLEMS = {
+  API_KEYS_WITHOUT_LICENSE: t`The embedding SDK is using API keys. This is intended for evaluation purposes and works only on localhost. To use on other sites, a license is required and you need to implement SSO.`,
```

Comment:
I don't think we should localize those messages, it would only make debugging and reporting issues harder

Reply:
Good point. I'll remove the localizations then.

Reply:
Fixed in https://github.com/metabase/metabase/pull/47821/commits/705ca68d684eb5c2f98b31bf4f72a99e8e7d233e, thanks!